### DISCUSSION TITLE
npm run tauri dev not deb

### DISCUSSION THREAD

**Discussion by sjqlwy** [freakydude/klipper-touch#19] [General]
Thank you for sharing!

**Comment by freakydude** [comment]
Thats true. I will fix that.
 - it's very very experimental at the moment. 
 
I tested all software aspects indivual - and it will work - but still stuck on the design itself ..

and at the moment: The framework itself works fine on my desktop - but not on bigtreetech manta + cb1 + tft35spi yet. I use the cb1 2.3.2 image - but the enabled the panfrost gpu first in 2.3.3 - thats my last try I think ;-) ...

What combination do you use?

**Comment by sjqlwy** [comment]
current on my x86 device with desktop monitor too, maybe you can attach your `/var/log/Xorg.0.log` file, as I have deal with klipperscreen with xorg error a lot with different arm device.
![image](https://github.com/freakydude/klipper-touch/assets/11326433/906099e9-5fc6-4b67-aefd-508640601065)

**Comment by freakydude** [comment]
Cool! 

Installing the BigTreeTech CB1 Image 2.3.3 with Mali Panfrost driver included works now. And I updated the distribution to Debian bookworm in the same move.

It does not seem like it is fully activated / not all bins are included... But anyways, good enough for the moment.

Got it to work.🙂

It's good to know you are watching. I can need feedback and help on the second turn after my ideas and open work items are settling down a little. 

Thank you, 
Best regards
